dev-docs/pb9-notes.md

```diff
@@ -0,0 +1,124 @@
+
+## Video ORTB2 Objects
+
+The Ortb2 core adapter utility no longer infers placement from context. Context = 'instream' now only refers to the technical integration method the publisher is using to interact with the player and is not relevant to the ortb2 bid requests. Adapters should not infer placement nor plcmt is instream from this value. Adapters are not permitted to only support placement and not plcmt; they are welcome to pass both. It is however reasonable to infer plcmt = 4 from context = outstream.
+
+## PAAPI
+
+Any module described as Fledge is now PAAPI. PAAPI Configuration is simplified and publishers experimenting with Fledge in 8.x should refer to the module documentation for updates.
+
+Publishers wishing to support PAAPI should install the PAAPI module and select the submodule appropriate for their top level seller (TLS). Both gpt.js and publisher designated top level decision logic have submodules. Ad servers wishing to add support for top level selling in Prebid.js may choose to issue instructions compatible with the newly released publisher designated TLS or submit their own submodule. 
+
+Bid adapters can now return either complete auction config or an `igb` object according to the Ortb community extension for PAAPI. Publishers wishing to be or to designate a component seller to handle the `igb` objects returned by some bid modules should configure PAAPI with, for example
+
+` pbjs.setConfig({
+    paapi: {
+       enabled: true,
+       componentSeller: {
+          auctionConfig: {
+                decisionLogicURL: 'publisher.example'
+                // ...
+          }
+       }
+    }
+ })`
+
+PAAPI for GPT now supports custom slot configuration. Also, the autoConfig option has been removed and replaced with configWithTargeting, defaulting to true, which sets GPT targeting and submits auction configs together. It differs in the previous autoconfig in that it no longer relies on gpt being available at time of requestBids, only at the time of setTargeting. 
+
+Publishers should be aware this behavior may prohibit submission of auction configuration to GPT sooner than the Prebid.js auction has completed, and will likely prefer to use `setPAAPIConfigForGPT`. We're hoping a futute gpt.js release will enable submission of configuration including unresolved promises earlier than the completion of the Prebid auction, by providing an `allConfigsSubmitted` type utility. Prebid support for other top level sellers will include this functionality in the near future.
+
+## Miscellaneous configuration changes (publishers):
+
+Pbadslot has had a deprecation warning issued, it is redundant specifying with imp.ext.gpid. In the future, the pre-auction module will not populate it and publishers setting pbadslot explicitly will see a warning. 
```

Comment:
the warning is there now, in the future we'll presumably remove support for it and the warning will go away with it.